horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/RunServiceImpl.java

```diff
@@ -458,16 +423,14 @@ public Response addRunFromData(String start, String stop, String test, String ow
             log.error("Failed to read data/metadata from upload file", e);
             throw ServiceException.badRequest("Provided data/metadata can't be read (JSON encoding problem?)");
         }
-        return addRunFromData(start, stop, test, owner, access, token, schemaUri, description, dataNode.toString(),
-                metadataNode);
+        return addRunFromData(start, stop, test, owner, access, schemaUri, description, dataNode.toString(), metadataNode);
     }
 
     @PermitAll // all because of possible token-based upload
```

Comment:
I think we need to restrict this to `@RolesAllowed(Roles.UPLOADER)` to limit the to users with that role